app/assets/javascripts/tenants.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
remove generated files that you don't use.

also read https://stackoverflow.com/questions/14045858/what-is-the-syntax-to-skip-creating-tests-assets-helpers-when-running-rails. You can skip this generation.